### DISCUSSION TITLE
Issue with Notepad++ item integration

### DISCUSSION THREAD

**Discussion by FroztySeven** [moudey/Shell#305] [General]
Hi, I've noticed an empty gap appearing past the "Properties" item while using this program. It only seems to happen with files that show the "Edit with Notepad++" item in the context menu.
Can this be fixed?, it's a little bothering.

![firefox_2023-08-03_01-10-03](https://github.com/moudey/Shell/assets/20623800/8837d517-7320-4dc7-a10d-02ce3c08f225)
![firefox_2023-08-03_01-11-13](https://github.com/moudey/Shell/assets/20623800/0ff161c5-04b6-426a-967d-98df4644fee3)

**Comment by RubicBG** [comment]
I can't give a solution to the problem, but you can solve it in another way:

Remove the "default" Notepad++ command and add a new command via Nilesoft Shell

**Reply by FroztySeven** [reply]
Thanks for your reply, yeah actually figured out i could just add: 
`item(type='file' where=this.disabled find='Edit with Notepad++' vis="remove")` to the static section in shell.nss. 
That does the trick 👍

**Comment by Alhlhli** [comment]
تم حل هذه المشكلة بالاصدار الجديد 1.9.15

This problem has been resolved in the new version 1.9.15
